setup.py

```diff
@@ -6,14 +6,15 @@
     author="Department for Levelling Up, Housing and Communities",
     description="Provides access to functionality common to creating a data dashboard.",
     name="gov_uk_dashboards",
-    version="4.4.3",
+    version="4.5.0",
     packages=find_packages(),
     install_requires=[
         "setuptools~=59.8.0",
         "dash~=2.0.0",
         "numpy>=1.22.0",
         "dash_bootstrap_components~=1.1.0",
         "pandas>=1.3",
+        "plotly~=5.5.0",
```

Comment:
Not a blocker, more a general thought.  It's a bit annoying that we need to define plotlys version 3 times in this repo, I wonder what other libraries do to prevent this.